blazor/rating/templates.md

```diff
@@ -9,17 +9,18 @@ documentation: ug
 
 # Templates in Blazor Rating Component
 
-The Blazor Rating component supports to customize the appearance of the rating item by using Templates.
+The Blazor Rating component allows you to customize the appearance of the rating items using templates. You can use templates to specify a custom layout for the rating items, which can include any content you want. This allows you to create a more customized and interactive rating experience for the user.
 
-The Template types of Rating are as follows:
+The template types of rating are as follows:
```

Comment:
The rating component supports below templates for item customization.

Reply:
updated as per comments.